lib/sass_spec/test.rb

```diff
@@ -8,7 +8,8 @@
 # Holder to put and run test cases
 class SassSpec::Test < Minitest::Test
   def self.create_tests(test_cases, options = {})
-    test_cases[0..options[:limit]].each do |test_case|
+    options[:limit] = test_cases.length + 1 unless options[:limit]
+    test_cases[0...options[:limit]].each do |test_case|
```

Comment:
`..` on Ruby is inclusive-inclusive, while `...` is inclusive...exclusive (which is what you'd expect `..` to be)